client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -83,14 +81,26 @@ export default function UserOperatorMultiStepForm({
   const isCasInternal =
     session?.user.app_role?.includes("cas") &&
     !session?.user.app_role?.includes("pending");
-
   if (isCasInternal || !userOperatorId || formSection) {
     return (
       <>
-        {isCasInternal && (
+        {isCasInternal && formData.is_new && formSection === 0 && (
+          <>
+            <Note message="This is a new operator. You must approve this operator before approving its admin." />
+            <UserOperatorReview
+              userOperator={formData as UserOperatorFormData}
+              userOperatorId={Number(userOperatorId)}
+              isOperatorNew={formData?.is_new}
+              operatorId={formData?.id}
+            />
+          </>
+        )}
+        {isCasInternal && formSection === 1 && (
```

Comment:
Just to confirm, should we only display review buttons to approve an admin request on the second page?

Reply:
If it's a existing operator, yes, we should only display the buttons on the second page.